features/search-replace.feature

```diff
@@ -1106,6 +1106,28 @@ Feature: Do global search/replace
       a:1:{i:0;O:10:"CornFlakes":0:{}}
       """
 
+  # Regression test for https://github.com/wp-cli/search-replace-command/issues/191
```

Comment:
This comment is unnecessary. It's simple enough to find a test's origin from `git blame`.

Reply:
removed!